### PR TITLE
Chore: Updates Changelog for 7.1.2

### PR DESCRIPTION
**Pull request by hugohaggmark**
### Features / Enhancements
  * **Explore**: Don't run queries on datasource change. [#26033](https://github.com/grafana/grafana/pull/26033), [@davkal](https://github.com/davkal)
  * **TemplateSrv**: Formatting options for ${__from} and ${__to}, unix seconds epoch, ISO 8601/RFC 3339. [#26466](https://github.com/grafana/grafana/pull/26466), [@torkelo](https://github.com/torkelo)
  * **Toolkit/Plugin**: throw an Error instead of a string. [#26618](https://github.com/grafana/grafana/pull/26618), [@leventebalogh](https://github.com/leventebalogh)

  ### Bug Fixes
  * **Dashbard**: Fix refresh interval settings to allow setting it to equal min_refresh_interval. [#26615](https://github.com/grafana/grafana/pull/26615), [@torkelo](https://github.com/torkelo)
  * **Explore**: Run queries on Metrics to Logs transition. [#26704](https://github.com/grafana/grafana/pull/26704), [@ivanahuckova](https://github.com/ivanahuckova)
  * **Flux**: Ensure connections to InfluxDB are closed. [#26735](https://github.com/grafana/grafana/pull/26735), [@sneddrs](https://github.com/sneddrs)
  * **Query history**: Fix search filtering if null value. [#26768](https://github.com/grafana/grafana/pull/26768), [@ivanahuckova](https://github.com/ivanahuckova)
  * **QueryOptions**: Fix not being able to change cache timeout setting. [#26614](https://github.com/grafana/grafana/pull/26614), [@torkelo](https://github.com/torkelo)
  * **StatPanel**: Fix stat panel display name not showing when explicitly set. [#26616](https://github.com/grafana/grafana/pull/26616), [@torkelo](https://github.com/torkelo)
  * **Templating**: Fixed access to system variables like __dashboard, __user & __org during dashboard load & variable queries. [#26637](https://github.com/grafana/grafana/pull/26637), [@torkelo](https://github.com/torkelo)
  * **TextPanel**: Fix content overflowing panel boundaries. [#26612](https://github.com/grafana/grafana/pull/26612), [@torkelo](https://github.com/torkelo)
  * **TimePicker**: Fix position and responsive behavior. [#26570](https://github.com/grafana/grafana/pull/26570), [@torkelo](https://github.com/torkelo)
  * **TimePicker**: Fixes app crash when changing custom range to nothing. [#26775](https://github.com/grafana/grafana/pull/26775), [@hugohaggmark](https://github.com/hugohaggmark)
  * **Units**: Remove duplicate SI prefix from mSv and µSv. [#26598](https://github.com/grafana/grafana/pull/26598), [@tofurky](https://github.com/tofurky)

✔ Generating changelog

### CODE REVIEW

**Review by aknuds1 — APPROVED** [review]

**Comment by hugohaggmark** [comment]
@aknuds1 are we thinking about not running the whole build pipeline for a commit with md files only?

**Comment by aknuds1** [comment]
@hugohaggmark Marcus started on a PR for CircleCI, but as regards Drone I think the main optimization is to try to implement caching (I think infra are working on supporting this). With caching, things should generally be a lot faster and we also don't run the risk of not running tests by accident.

**Comment by hugohaggmark** [comment]
> @hugohaggmark Marcus started on a PR for CircleCI, but as regards Drone I think the main optimization is to try to implement caching (I think infra are working on supporting this). With caching, things should generally be a lot faster and we also don't run the risk of not running tests by accident.

Great, thank you

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
+# 7.1.2 (2020-08-05)
+
+### Features / Enhancements
+  * **Explore**: Don't run queries on datasource change. [#26033](https://github.com/grafana/grafana/pull/26033), [@davkal](https://github.com/davkal)
+  * **TemplateSrv**: Formatting options for ${__from} and ${__to}, unix seconds epoch, ISO 8601/RFC 3339. [#26466](https://github.com/grafana/grafana/pull/26466), [@torkelo](https://github.com/torkelo)
+  * **Toolkit/Plugin**: throw an Error instead of a string. [#26618](https://github.com/grafana/grafana/pull/26618), [@leventebalogh](https://github.com/leventebalogh)
+
+  ### Bug Fixes
+  * **Dashbard**: Fix refresh interval settings to allow setting it to equal min_refresh_interval. [#26615](https://github.com/grafana/grafana/pull/26615), [@torkelo](https://github.com/torkelo)
+  * **Explore**: Run queries on Metrics to Logs transition. [#26704](https://github.com/grafana/grafana/pull/26704), [@ivanahuckova](https://github.com/ivanahuckova)
```
Comment (ivanahuckova) [inline CHANGELOG.md:10]:
```suggestion
```

I would remove this as the bug is introduced and fixed in the same release - therefore the end users don't have to be aware of it/it has no impact on them.

**Review by ivanahuckova — APPROVED** [review]
I have added 1 suggestion, but other than that LGTM! 🕺

**Pull request merged**